util/fbsetroot.cc

```diff
@@ -219,8 +219,8 @@ void fbsetroot::modula(int x, int y) {
     char data[32];
     long pattern = 0;
 
-    register int i;
-
+    /* register */ int i; // register storage class deprecated C++11,
```

Comment:
ya, really remove `register` fully

Reply:
(Moot for the new version of this PR when it happens, but yes, I did that in #76, thanks.)